includes/emails/class-give-email-tags.php

```diff
@@ -55,14 +55,16 @@ class Give_Email_Template_Tags {
 	 * @param string   $description Email tag description text
 	 * @param callable $func        Hook to run when email tag is found
 	 * @param string   $context     Email tag category
+	 * @param boolean  $is_admin    Set true to show the tag only for admin context.
 	 */
-	public function add( $tag, $description, $func, $context = '' ) {
+	public function add( $tag, $description, $func, $context = '', $is_admin = false ) {
```

Comment:
@Sidsector9 changes which you have done in `give_add_email_tag` function should be in this function. Let me know if you have any question